include/grl/kuka/KukaJAVAdriver.hpp

```diff
@@ -350,10 +329,75 @@ namespace grl { namespace robot { namespace arm {
               }
               std::cout << "\n";
           }
-        
-          kukaJavaDriverP->async_send_flatbuffer(fbbP);
-        }
-       
+
+          int ret;
+          ret = sendto(socket_local, fbbP->GetBufferPointer(), fbbP->GetSize(), 0, (struct sockaddr *)&dst_sockaddr, sizeof(dst_sockaddr));
+          if (ret != fbbP->GetSize())
+              printf("Send Error in masterJoint: ret = %d, len = %u\n", ret, fbbP->GetSize());
+
+
+              // Receiving data from Sunrise
+
+              int num;
+              temp_mask = mask;
+
+              struct timeval tv;
+              tv.tv_sec = 0;
+              tv.tv_usec = 0;
+
+              num = select(FD_SETSIZE, &temp_mask, &dummy_mask, &dummy_mask, &tv);
+
+                    if (num > 0)
+                    {
+                              if (FD_ISSET(socket_local, &temp_mask))
+                              {
+
+                                   unsigned char recbuf[1024];
```

Comment:
This is the receiving buffer, which is only in this scope.. we take the useful data out of it and put it in our own member variables